spec/requests/api/moves_controller_filter_spec.rb

```diff
@@ -32,17 +32,17 @@
 
     describe 'by supplier_id' do
       let(:filter_params) { { filter: { supplier_id: supplier.id } } }
-      let(:expected_moves) { create_list :move, 2, supplier: supplier }
-      let(:unexpected_moves) { create_list :move, 2, supplier: alternative_supplier }
+      let(:expected_moves) { create_list :move, 1, supplier: supplier }
+      let(:unexpected_moves) { create_list :move, 1, supplier: alternative_supplier }
```

Comment:
I like these kinds of changes, the only things is that it
sounds a bit weird using `create_list` with 1 element,
I suppose it is because it returns an array,
the alternative would be to wrap it in an array 🤔 
`[ create(:move, supplier: supplier ) ]`

Not very opinionated though

Reply:
I guess creating a list with one element is twice as fast as creating it with two elements 🤔

Reply:
my "dilemma" was more about
`let(:expected_moves) { create_list :move, 1, supplier: supplier }`
versus the equivalent form:
`let(:expected_moves) { [ create(:move, supplier: supplier) ] }`  
but that's fine, not very important.

Reply:
potatohs vs potatoes :-)